Sourcery/Models/Method.swift

```diff
@@ -150,11 +150,23 @@ public final class Method: NSObject, SourceryModel, Annotated {
     /// Whether method is a failable initializer
     public let isFailableInitializer: Bool
 
-    /// Whether method is a convenience initializer
+    // sourcery: skipEqaulitey, skipDescription, skipCoding, skipJSExport
+    /// :nodoc:
+    @available(*, deprecated: 0.7, message: "Use isConvenienceInitializer instead")
     public var isConvenienceInitialiser: Bool {
         return attributes[Attribute.Identifier.convenience.name] != nil
     }
 
+    /// Whether method is a convenience initializer
+    public var isConvenienceInitializer: Bool {
```

Comment:
Changed here from `s` to `z` as we use `z` in two other properties, but I'm ok with changing `z` to `s` too, @krzysztofzablocki let me know if you prefer other option =)